.travis.yml

```diff
@@ -4,84 +4,72 @@ matrix:
   include:
     - os: osx
       env:
-        - TRAVIS_PYTHON_VERSION="2.7"
+        - TRAVIS_PYTHON_VERSION="3.6"
         - DEPLOYABLE="true"
     - os: osx
       env:
-        - TRAVIS_PYTHON_VERSION="3.5"
+        - TRAVIS_PYTHON_VERSION="3.7"
         - DEPLOYABLE="true"
     - os: osx
       env:
-        - TRAVIS_PYTHON_VERSION="3.6"
+        - TRAVIS_PYTHON_VERSION="3.8"
+        - DEPLOYABLE="true"
+    - os: osx
+      env:
+        - TRAVIS_PYTHON_VERSION="3.9"
         - DEPLOYABLE="true"
+    - os: osx
+      env:
+        - TRAVIS_PYTHON_VERSION="3.10"
+        - DEPLOYABLE="true"
+    - os: linux
+      env:
+        - TRAVIS_PYTHON_VERSION="3.6"
     - os: linux
       env:
-        - TRAVIS_PYTHON_VERSION="2.7"
+        - TRAVIS_PYTHON_VERSION="3.7"
     - os: linux
       env:
-        - TRAVIS_PYTHON_VERSION="3.5"
+        - TRAVIS_PYTHON_VERSION="3.8"
     - os: linux
       env:
-        - TRAVIS_PYTHON_VERSION="3.6"
-    #- services: docker
-    #  sudo: required
-    #  env:
-    #    - DEPLOY_TARGET="manylinux1"
-    #    - DEPLOYABLE="true"
+        - TRAVIS_PYTHON_VERSION="3.9"
+    - os: linux
+      env:
+        - TRAVIS_PYTHON_VERSION="3.10"
 
 dist: trusty
 
 before_install:
     - |
       # Skip if manylinux1
-      if [ "$DEPLOY_TARGET" = "manylinux1" ]; then
```

Comment:
@Carloscerq Seems like you have removed the full `manylinux` setup which is needed to build cross distribution linux wheels.

Reply:
@ibayer was it being used. I figured out since the declaration on the `matrix` was commented, it wasn't being used. Should I add it?

Reply:
@Carloscerq 

Since it's `before_install:` it should always run. Also even if it hasn't been run with the current settings it needs to run in the future to get valid manylinux wheels.

@AlexJoz Am I missing something?

Reply:
@ibayer seems like you've actually disabled manylinux build [c6690f](https://github.com/ibayer/fastFM/commit/c6690f3fe07574967e89c395f470b6484a158283). But sorry I don't remember what's the case here, as we used to work in gitlab repo for the most recent fastfm staff ^^.

Reply:
To re-enable building wheel, I'd recommend to move cibuildwheel https://github.com/pypa/cibuildwheel
Also, we may be able to consider moving to GitHub Action.

Reply:
if/ I'm not mistaken, we had some actions in fastfm2. I believe we can borrow some code form there, if needed

Reply:
@Carloscerq 
I stand corrected, it probably doesn't make sense to reactivate `manylinux` with travis.

@chezou Thanks for you input! I think we are actually using such a setup with https://github.com/palaimon/fastfm2.
I was original hoping fastfm2 would replace fastfm quickly but more urgent stuff keeps popping up. :-/

Reply:
Totally understandable your situation 😄 
I looked at a glance for fastfm2 repo, and it was what I intended 👍  https://github.com/palaimon/fastfm2/blob/master/.github/workflows/wheels.yaml